features/mbedtls/targets/TARGET_NUVOTON/TARGET_M480/sha/sha512_alt_sw.c

```diff
@@ -24,21 +24,15 @@
  *  http://csrc.nist.gov/publications/fips/fips180-2/fips180-2.pdf
  */
 
-#if !defined(MBEDTLS_CONFIG_FILE)
-#include "mbedtls/config.h"
-#else
-#include MBEDTLS_CONFIG_FILE
-#endif
+#include "mbedtls/sha512.h"
 
 #if defined(MBEDTLS_SHA512_C)
 #if defined(MBEDTLS_SHA512_ALT)
 
-#include "mbedtls/sha512.h"
-
 #if defined(_MSC_VER) || defined(__WATCOMC__)
```

Comment:
I think this is just for windows, so not needed in mbed OS

Reply:
@andresag01 Same as above.